lib/transformer.dart

```diff
@@ -65,12 +69,23 @@ class InitializeTransformer extends Transformer {
     if (newEntryPoint == null) {
       newEntryPoint = entryPoint.replaceFirst('.dart', '.bootstrap.dart');
     }
-    return new InitializeTransformer(entryPoint, newEntryPoint);
+    var htmlEntryPoint = settings.configuration['htmlEntryPoint'];
```

Comment:
we could do the change from `'htmlEntryPoint'` -> `'html_entry_point'` right away or separately (issue #14)